packs/data/equipment.db/bottled-sunlight-moderate.json

```diff
@@ -21,7 +21,7 @@
             "die": "d4"
         },
         "description": {
-            "value": "<p><strong>Activate</strong> <span class=\"pf2-icon\">1</span> Interact</p>\n<p>This mirrored bottle contains various chemicals dissolved in two immiscible solvents. Shaking the bottle induces chemical reactions that cause it to glow. For 1 hour, the bottle sheds bright light in a 20-foot radius (and dim light to the next 40 feet).</p>\n<p>During this hour, it can be thrown like a bomb. This requires a single action to Activate with a Strike, like other bombs, and deals the damage below. If it's not thrown within an hour of the first activation, it defuses harmlessly and is consumed.</p>\n<p>The bomb deals 2d4 positive damage and 2 positive splash damage, as well as 2d4 additional fire damage. As normal, positive damage harms only undead and other creatures with negative healing. Undead with a particular vulnerability to sunlight (such as vampires or wraiths) take the maximum amount of positive damage from bottled sunlight (2 instead of 2d4). For any ability that depends on the number of weapon dice bottled sunlight has, count only the positive damage dice.</p>"
+            "value": "<p><strong>Activate</strong> <span class=\"pf2-icon\">1</span> Interact</p>\n<p>This mirrored bottle contains various chemicals dissolved in two immiscible solvents. Shaking the bottle induces chemical reactions that cause it to glow. For 1 hour, the bottle sheds bright light in a 20-foot radius (and dim light to the next 40 feet).</p>\n<p>During this hour, it can be thrown like a bomb. This requires a single action to Activate with a Strike, like other bombs, and deals the damage below. If it's not thrown within an hour of the first activation, it defuses harmlessly and is consumed.</p>\n<p>The bomb deals 2d4 positive damage and 2 positive splash damage, as well as 2d4 additional fire damage. As normal, positive damage harms only undead and other creatures with negative healing. Undead with a particular vulnerability to sunlight (such as vampires or wraiths) take the maximum amount of positive damage from bottled sunlight (8 instead of 2d4). For any ability that depends on the number of weapon dice bottled sunlight has, count only the positive damage dice.</p>"
```

Comment:
```suggestion
            "value": "<p><strong>Activate</strong> <span class=\"pf2-icon\">1</span> Interact</p>\n<p>This mirrored bottle contains various chemicals dissolved in two immiscible solvents. Shaking the bottle induces chemical reactions that cause it to glow. For 1 hour, the bottle sheds bright light in a 20-foot radius (and dim light to the next 40 feet).</p>\n<p>During this hour, it can be thrown like a bomb. This requires a single action to Activate with a Strike, like other bombs, and deals the damage below. If it's not thrown within an hour of the first activation, it defuses harmlessly and is consumed.</p>\n<p>The bomb deals 2d4 positive damage and 2 positive splash damage, as well as 2d4 additional fire damage. As normal, positive damage harms only undead and other creatures with negative healing. Undead with a particular vulnerability to sunlight (such as vampires or wraiths) take the maximum amount of positive damage from bottled sunlight ([[/r 8[positive]]] instead of 2d4). For any ability that depends on the number of weapon dice bottled sunlight has, count only the positive damage dice.</p>"
```